src/features/island/fruit/FruitSeedling.tsx

```diff
@@ -39,10 +39,14 @@ export const FruitSeedling: React.FC<Props> = ({ fruitName, timeLeft }) => {
   const isAlmostReady = growPercentage >= 50;
   const isHalfway = growPercentage >= 25 && !isAlmostReady;
 
-  const isBanana = fruitName === "Banana";
-  const description = isBanana
-    ? `Bananas Growing`
-    : `${fruitName} Tree Growing`;
+  const description =
+    fruitName === "Banana"
+      ? `Bananas Growing`
+      : fruitName === "Blueberry"
+        ? `Blueberry Bush Growing`
+        : fruitName === "Tomato" || fruitName === "Lemon"
+          ? `${fruitName} Plant Growing`
+          : `${fruitName} Tree Growing`;
```

Comment:
It might be time to abstract this to a helper function.

Example:

```ts
const getFruitDescription = (fruitName) => {
  if (fruitName === "Banana") return "Bananas Growing"
  if (fruitName === "BlueBerry").....
  
}
```

Reply:
```typescript
  const description =
    fruitName === "Banana" || fruitName === "Tomato" || fruitName === "Lemon"
      ? `${fruitName} Plant Growing`
      : `${fruitName} Tree Growing`;
```
Actually I can combine Banana with Tomato and Lemon for the fruit name plant description, what do you think?

Reply:
```typescript
  const description =
    fruitName === "Banana" || fruitName === "Tomato" || fruitName === "Lemon"
      ? `${fruitName} Plant Growing`
      : `${fruitName} Tree Growing`;
```
@c-py Actually I can combine Banana with Tomato and Lemon for the fruit name plant description, what do you think?

Reply:
```typescript
  switch (fruitName) {
    case "Banana":
    case "Tomato":
    case "Lemon":
      description = `${fruitName} Plant Growing`;
      break;
    case "Blueberry":
      description = "Blueberry Bush Growing";
      break;
    default:
      description = `${fruitName} Tree Growing`;
  }
  ```
@c-py If i use a switch statement like this how does it sound?